src/domain/entity/phpSetting.go

```diff
@@ -19,3 +24,44 @@ func NewPhpSetting(
 		Options: options,
 	}
 }
+
+// format: name:value:option,option,option
```

Comment:
The name option is too generic, we should have used suggestedValues or something like that. Since modifying that now will require a frontend change, let's just adjust this comment from option,option,option to suggestedValue1, suggestedValue2, suggestedValue3. That way we can at least know what's this is about.

Reply:
Done!